### ISSUE TITLE
Replace alias variables with <Alias> elements

### ISSUE THREAD

**Issue by t-sommer** [modelica/fmi-standard#807]
As discussed at the Berlin 2020 Design Meeting (meeting minutes at https://github.com/modelica/fmi-design/blob/master/Meetings/2020-02-25-FMI-Design-Berlin/minutes.adoc)

Proposal:

The aliases are only information attached to a variable (no actual variable). It's up to the importer howto handle (display, log, etc.) them.

```xml
<Float64 name="torque.tau_0" valueReference="2" description="Accelerating torque acting at flange"
      declaredType="Modelica.SIunits.Torque">
   <Alias name="torque.tau_1" description="Alias 1" displayUnit="kN.m"/>
   <Alias name="torque.tau_2" description="Alias 2" displayUnit="nN.m"/>
</Float64>
```

Only the following attributes are allowed for `<Alias>` elements:

- `name`
- `description`
- `display unit`

All attributes follow the same conventions as for variables.

**Comment by chrbertsch** [comment]
Create also non-normative text about what the exporting tools have to do (e.g. find the common range of min, max, ....)

**Comment by CSchulzeTLK** [comment]
* There is one variable, and aliases (additional names with varied displayUnit and description) which belong to them
* The namespace for ModelVariables and aliases is merged. So there must not be an alias which has the same name as a variable. 
* just the variable can be set with `set{VariableType}`, because its the only one with a valueReference
* Alias of time variable? No restriction necessary.
* if causality is
  * parameter
    * the aliases are treated similar to a calculatedParameter
  * input/output
    * also possible, no reason to restrict
* clock variables don't have aliases? => crosscheck the clocks section
* all attributes are equal, except:
  * name
  * displayUnit
  * description
* i.e. all are treated as if they e.g. had the same `<Unit name>` and the same `<Unit><BaseUnit>` definitions.
* exporting tool has to merge the other attributes
  * min/max/nominal/quantity/unit/variability...
  * no longer separate min max values (orignially allowed to provide submodel information in systems with connected submodels)
  * no longer contraticting start values for constants
* it is obvious which parameter name should be displayed in the importing tool

=> replace the word "aliasing" in the rest of the specification

**Comment by chrbertsch** [comment]
Design Meeting: 

if causality is
    parameter
        the aliases are treated similar to a calculatedParameter

--> this is true but shall not be explicitly written.

clocks could have aliases with the new concept

If an alias allowed to have a different quantity or only a different unit (e.g. "m" and "s") --> No
Reference  the section for displayunits.

--> Agreement

**Comment by clagms** [comment]
Working on a PR for this now.

**Comment by chrbertsch** [comment]
fixed with #834
